doc/classes/PackedByteArray.xml

```diff
@@ -449,24 +475,29 @@
 			<return type="bool" />
 			<param index="0" name="right" type="PackedByteArray" />
 			<description>
+				Returns [code]true[/code] if contents of the arrays differ.
 			</description>
 		</operator>
 		<operator name="operator +">
 			<return type="PackedByteArray" />
 			<param index="0" name="right" type="PackedByteArray" />
 			<description>
+				Returns a new [PackedByteArray] with contents of [param right] added at the end of this array. For better performance, consider using [method append_array] instead.
 			</description>
 		</operator>
 		<operator name="operator ==">
 			<return type="bool" />
 			<param index="0" name="right" type="PackedByteArray" />
 			<description>
+				Returns [code]true[/code] if contents of both arrays are the same, i.e. they have all equal bytes at the corresponding indices.
 			</description>
 		</operator>
 		<operator name="operator []">
 			<return type="int" />
 			<param index="0" name="index" type="int" />
 			<description>
+				Returns the byte at index [param index]. Using index out of array's bounds will result in an error.
```

Comment:
Note that negative indices are also supported, valid index is in [`-size()`, `size() - 1`] range.
The same for other packed arrays too.